commcare_connect/users/signals.py

```diff
@@ -6,7 +6,9 @@
 
 @receiver(user_signed_up)
 def create_org_for_user(request, user, **kwargs):
-    if not user.members.exists():
```

Comment:
@snopoke This check was failing with AttributeError for members on the user object. I wrapped this in a try-except. I am not totally sure if this is a good solution for this.

Reply:
Thanks Fixed. [6c3f65f](https://github.com/dimagi/commcare-connect/pull/60/commits/6c3f65f5f29ad456da4622baefb3c72ae45347e6)